wolfcrypt/src/ecc.c

```diff
@@ -6931,6 +6931,14 @@ int wc_ecc_import_point_der_ex(byte* in, word32 inLen, const int curve_idx,
                     (ECC_CURVE_FIELD_PRIME | ECC_CURVE_FIELD_AF |
                         ECC_CURVE_FIELD_BF));
 
+            /* validate prime is prime for custom curves */
```

Comment:
Please exclude the code based on a check like: !ECC_TRUSTED_CUSTOM_CURVE

Reply:
Good change!
Still would like to be able to exclude if the customer explicitly trusts the curve.

Reply:
Ok, will guard it on the WOLFSSL_VALIDATE_ECC_IMPORT macro. Often that is off by default....but the macro name I think makes since here since it is also adding in checking on the ECC key when imported and has been used in similar situations.